.github/workflows/release.yaml

```diff
@@ -12,13 +12,6 @@ jobs:
           - os: ubuntu-18.04
```

Comment:
Is there a reason to leave this as a matrix?

Reply:
erm, maybe when we want to have more archs